compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -32,6 +32,19 @@ void print_version(void)
   std::cout << vconone::get_copyright() << std::endl;
 }
 
+int handleAutoAlgorithm(arser::Arser &arser, mpqsolver::bisection::BisectionSolver &solver)
+{
+  solver.algorithm(mpqsolver::bisection::BisectionSolver::Algorithm::Auto);
+  auto data_path = arser.get<std::string>("--visq_data");
+  if (data_path.empty())
+  {
+    std::cerr << "ERROR: for auto mode you must provide valid visq_file" << std::endl;
```

Comment:
Q) above option checks `visq_data` but message is `visq_file`. is this intended?

Reply:
no. i'll change it to visq_file